scriptmodules/supplementary/skyscraper.sh

```diff
@@ -0,0 +1,696 @@
+#!/usr/bin/env bash -x
```

Comment:
please remove the +x as pointed out by @hhromic

Reply:
Done, thanks for noticing @hhromic.